specification/azsadmin/resource-manager/storage/Microsoft.Storage.Admin/preview/2016-05-01/storageaccounts.json

```diff
@@ -0,0 +1,441 @@
+                }
+            }
+        }
+    },
+    "definitions": {
+        "StorageAccountState": {
+            "readOnly": true,
+            "description": "Storage account state.",
+            "type": "string",
+            "enum": [
+                "Creating",
+                "Succeeded",
+                "Suspended"
+            ],
+            "x-ms-enum": {
+                "name": "StorageAccountState",
+                "modelAsString": true
+            }
+        },
+        "StorageAccountType": {
+            "description": "Storage account type.",
+            "type": "string",
+            "enum": [
+                "Standard_LRS",
+                "Standard_GRS",
+                "Standard_ZRS",
+                "Standard_RAGRS",
+                "Premium_LRS"
+            ],
+            "x-ms-enum": {
+                "name": "StorageAccountType",
+                "modelAsString": true
+            }
+        },
+        "LocationStatus": {
+            "readOnly": true,
+            "type": "string",
+            "description": "Gets the status indicating whether the primary location of the storage account is available or unavailable.",
+            "enum": [
+                "Available",
+                "Unavailable"
+            ],
+            "x-ms-enum": {
+                "name": "LocationStatus",
+                "modelAsString": true
+            }
+        },
+        "StorageAccountOperation": {
+            "readOnly": true,
+            "description": "The permissions of storage account in WAC.",
+            "type": "string",
+            "enum": [
+                "None",
+                "Create",
+                "Update",
+                "Suspend",
+                "Delete",
+                "Undelete"
+            ],
+            "x-ms-enum": {
+                "name": "StorageAccountOperation",
+                "modelAsString": true
+            }
+        },
+        "StorageAccountStatus": {
+            "description": "The state of storage account in WAC.",
+            "type": "string",
+            "enum": [
+                "Active",
+                "Deleted",
+                "OutOfRetentionPeriod",
+                "Recycled"
+            ],
+            "x-ms-enum": {
+                "name": "StorageAccountStatus",
+                "modelAsString": true
+            }
+        },
+        "WacAccountPermissions": {
+            "description": "Current operation being performed on Storage Account",
+            "type": "string",
+            "enum": [
+                "Empty",
+                "None",
+                "Read",
+                "Delete",
+                "Write",
+                "Full"
+            ],
+            "x-ms-enum": {
+                "name": "WacAccountPermissions",
+                "modelAsString": true
+            }
+        },
+        "WacAccountStates": {
+            "description": "Current operation being performed on Storage Account",

```

Comment:
can you double check description?

Reply:
same comment as above, should we use the name `StorageAccountStates` here, and avoid using acronym `wac`.